### DISCUSSION TITLE
scroll to top button on UI

### DISCUSSION THREAD

**Discussion by shalva97** [minbrowser/min#1494] [Ideas]
for some websites its often inconvenient to drag scroll bar all the way to top.

A scroll to top button, which would also work with Fits's law, would be awesome. preferably at top right corner, but of course, there are other buttons in  that corner. but who needs title bar buttons right? I mean, when you open a program there is so little chance to press maximize button twice, or press close button more than once...

**Comment by PalmerAL** [comment]
Maybe not exactly what you're looking for, but if you install the [vim-mode userscript](https://github.com/PalmerAL/min-vim-mode), you get a "scroll to top" keyboard shortcut.

**Reply by shalva97** [reply]
nope, I hate everything related to vim. 

I had 2 buttons without function on my G300S mouse, so decided to add some macros. Now I can scroll top and bottom at a single mouse click.
